c++/src/kj/compat/tls.c++

```diff
@@ -587,10 +567,8 @@ public:
     //   So, we make some copies here.
     auto& tlsRef = tls;
     auto hostnameCopy = kj::str(hostname);
-    return inner->connect().then(
-        [&tlsRef,hostname=kj::mv(hostnameCopy)](Own<AsyncIoStream>&& stream) {
-      return tlsRef.wrapClient(kj::mv(stream), hostname);
-    });
+    auto stream = co_await inner->connect();
+    co_return co_await tlsRef.wrapClient(kj::mv(stream), hostname);
```

Comment:
Probably should be hostnameCopy instead of hostname here?

Reply:
I don't think we need hostnameCopy anymore at all actually.